upi/openstack/02_network.yaml

```diff
@@ -19,13 +19,17 @@
     command:
       cmd: "openstack network set --tag {{ cluster_id_tag }} {{ os_network }}"
 
+  - name: 'Create the cluster address-scope'
```

Comment:
This two tasks can probably be moved to the 021_network.yaml and again include that file only for 
when: os_networking_type == "CiscoACI"